src/bundle/Resources/views/themes/admin/content/edit_base.html.twig

```diff
@@ -62,7 +62,7 @@
                     </div>
                 {% else %}
                     <div class="row">
-                        <div class="offset-1 col-6 p-0">
+                        {% if not is_full_width|default(false) %}<div class="offset-1 col-6 p-0">{% endif %}
```

Comment:
Are you sure we want to complemtely remove this div? Instead of adding class `col-12` or sth like this?

Reply:
Yeah, on the second thought I think it may be better to only change CSS classes.